crates/anvil/core/src/eth/transaction/optimism.rs

```diff
@@ -145,7 +328,7 @@ impl DepositTransaction {
     /// - `input`
     pub fn decode_inner(buf: &mut &[u8]) -> Result<Self, DecodeError> {
         Ok(Self {
-            nonce: Decodable::decode(buf)?,
+            nonce: U256::from(0),
```

Comment:
Also was this wrong?

```suggestion
            nonce: U256::ZERO,
```